.gitlab-ci.yml

```
@@ -784,6 +784,32 @@ zombienet-tests-malus-dispute-valid:
   tags:
     - zombienet-polkadot-integration-test
 
+zombienet-tests-deregister-register-validator:
+  stage:                           stage3
+  image:                           "docker.io/paritytech/zombienet:v1.2.42"
+  <<:                              *kubernetes-env
+  <<:                              *zombienet-refs
+  needs:
+    - job:                         publish-polkadot-debug-image
```

```suggestion
    - job:                         publish-polkadot-debug-image
      artifacts:                 true
```
It's better to set this variable explicitly